blt/settings.py

```diff
@@ -511,11 +522,17 @@
 
 # Twitter
 
-BEARER_TOKEN = os.environ.get("BEARER_TOKEN")
-APP_KEY = os.environ.get("APP_KEY")
-APP_KEY_SECRET = os.environ.get("APP_KEY_SECRET")
-ACCESS_TOKEN = os.environ.get("ACCESS_TOKEN")
-ACCESS_TOKEN_SECRET = os.environ.get("ACCESS_TOKEN_SECRET")
+# BEARER_TOKEN = os.environ.get("BEARER_TOKEN")
+# APP_KEY = os.environ.get("APP_KEY")
+# APP_KEY_SECRET = os.environ.get("APP_KEY_SECRET")
+# ACCESS_TOKEN = os.environ.get("ACCESS_TOKEN")
+# ACCESS_TOKEN_SECRET = os.environ.get("ACCESS_TOKEN_SECRET")
```

Comment:
This needs to be reverted. You should have used envs for the secrets instead of exposing them.

Reply:
Oh sorry didnt see them I'll revert them these are not part of this project they are part of twitter project which we have earlier integrated